pyfarm/master/api/tasklogs.py

```diff
@@ -163,7 +163,7 @@ def post(self, job_id, task_id, attempt):
                            error="Specified task not found"), NOT_FOUND
 
         path = join(LOGFILES_DIR, g.json["identifier"])
-        if not realpath(path).startswith(LOGFILES_DIR):
+        if not abspath(path).startswith(LOGFILES_DIR):
```

Comment:
What's the reason for this change?

(Also, again, this doesn't really look like it belongs in here.)